.neuro/live-s3.yml

```
@@ -0,0 +1,314 @@
+    name: $[[ flow.title ]]-label-studio
+    http_port: 443
+    http_auth: False
+    life_span: 1d
+    pass_config: True
+    detach: False
+    browse: True
+    volumes:
+      - secret:gh-rsa:/root/.ssh/id-rsa
+    env:
+      PROJECT: /usr/project
+      DATA_PATH: /usr/project/data/Images
+      LABEL_STUDIO_LOCAL_FILES_SERVING_ENABLED: True
+      LABEL_STUDIO_PROJECT: /label-studio-project
+      LS_TOKEN: secret:ls-token
+      LS_PORT: 443
+    bash: |
+      git clone -b ${PROJECT_GIT_BRANCH} ${PROJECT_GIT_REPO} ${PROJECT}
+
+      # Pull from S3
+      neuro blob cp blob:${BUCKET_NAME}/label_studio.sqlite3 "${PROJECT}/data/label_studio.sqlite3"
+
+
+      COMPLETIONS=${LABEL_STUDIO_PROJECT}/completions
+
+      if [[ -f "${PROJECT}/data/label_studio.sqlite3" ]];
+      then
+        echo Restoring existing label studio data
+        cp ${PROJECT}/data/label_studio.sqlite3 /label-studio/data/label_studio.sqlite3
+      else
+        echo Initializing label studio
+        label-studio init dogs --username test@neu.ro --password 12345678 \
+                --user-token ${LS_TOKEN}  \
+                --label-config ${PROJECT}/label_studio/LabelConfig.xml
+      fi
+
+      export S3_CREDENTIALS=$(neuro blob statcredentials $CREDENTIALS)
+      export BUCKET=$(neuro blob statcredentials $CREDENTIALS | grep bucket_name | awk '{print $2}')
+      export REGION=$(neuro blob statcredentials $CREDENTIALS | grep region_name | awk '{print $2}')
+      export ENDPOINT=$(neuro blob statcredentials $CREDENTIALS | grep endpoint_url | awk '{print $2}')
+      export KEY_ID=$(neuro blob statcredentials $CREDENTIALS | grep access_key_id | awk '{print $2}')
+      export KEY=$(neuro blob statcredentials $CREDENTIALS | grep secret_access_key | awk '{print $2}')
+
+      echo "Starting label-studio"
+      python3 ${PROJECT}/label_studio/launch_ls.py \
+        --project_root ${PROJECT} \
+        --bucket ${BUCKET} \
+        --region_name ${REGION} \
+        --s3_endpoint ${ENDPOINT} \
+        --aws_access_key_id ${KEY_ID} \
+        --aws_secret_access_key ${KEY} \
+        -- \
+        start dogs --port ${LS_PORT} --no-browser
+
+
+      # Push labeling completions (label studio-specific files) and annotation results
+      neuro blob cp ${PROJECT}/data/result.json blob:${BUCKET_NAME}
+      neuro blob cp /label-studio/data/label_studio.sqlite3 blob:${BUCKET_NAME}
+
+      # Validate dataset:
+      neuro blob ls -r blob:${BUCKET_NAME}
+      echo "Total images: $(neuro blob ls -r blob:${BUCKET_NAME} | grep -oP '\w*\.\w+$' | wc -l)" # grep files only
+
+  deploy_inference_platform:
+    image: $[[ images.seldon.ref ]]
+    name: $[[ flow.title ]]-test-inference
+    preset: gpu-small
+    http_port: 5000
+    http_auth: False
+    life_span: 5h
+    detach: True
+    params:
+      run_id:
+        descr: MLFlow run ID, which model should be deployed
+      mlflow_storage:
+        descr: Storage path, where MLFlow server stores model binaries
```

WDYT if we add a dedicated volume for MLFlow artifacts and use it here as the default parameter value?

```suggestion
      mlflow_storage:
        descr: Storage path, where MLFlow server stores model binaries
        default: ${{ volumes.mlflow_storage.ref_rw }}
```

---

This will just move complexity to the code as instead of using fixed local fs path (`/storage/model.h5`) we will have to pass `run_id` to the code and look for model in `/mlruns/${RUN_ID}/artifacts/model/data/model.h5`.

I can do it if you think it's neccessary, but I'm not sure it helps :)